spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -265,7 +265,7 @@ Embedded objects can be stored in the values of embedded Maps.
 Collections of embedded objects can also be stored as the values of embedded Maps.
 
 
-==== Relationships
```

Comment:
I would keep a higher-level section called `Relationships` and just summarize all types of relationships that are supported. I believe we have three types: one-to-many via `@Descendants`, one-to-many or one-to-one using `@Reference`, and embedded entities. WDYT?

Reply:
Makes sense.  I will reorganize.